releng/releng-target/lsp4j.target.target

```diff
@@ -6,8 +6,8 @@
 			<repository location="https://download.eclipse.org/tools/orbit/simrel/orbit-aggregation/2023-09"/>
 			<unit id="com.google.gson" version="2.10.1"/>
 			<unit id="com.google.gson.source" version="2.10.1"/>
-			<unit id="com.google.guava" version="32.1.2.jre"/>
```

Comment:
i assume you need to split this up into two orbit locations

Reply:
https://github.com/eclipse-lsp4j/lsp4j/compare/cd_widenGuava

Reply:
Why do you think that the orbit should be split up here? I want to make sure I don't mess things up in 0.21.2 that I am working on.

Reply:
For sure the new version of guava needs 2024-03 to be added...

Reply:
I think I answered the question. It is about the renaming of the jakarta.websocket bundles?

Reply:
Yes, I think these 

![image](https://github.com/eclipse-lsp4j/lsp4j/assets/208716/8bc24bfc-e20a-4018-bbf4-284f4aa66ef7)

Reply:
Thanks Ed. I think something is going wrong here as we don't use the older jakarta.websocket, but instead jakarta.websocket-api in the code, so I will look at that.